Makefile

```diff
@@ -199,3 +203,13 @@ $(LICHE):
 
 $(PROMU):
 	$(call fetch_go_bin_version,github.com/prometheus/promu,$(PROMU_VERSION))
+
+$(PROTOC):
+	@echo ">> fetching protoc@${PROTOC_VERSION}"
+	@if [ ! -x '$(BIN_DIR)/protoc' ]; then \
```

Comment:
Why is this? makefile already checks if command in variable  `$(PROTOC)`  exists. Let's remove it to fetch it always, I think to simplify.